src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  width: 7 * $day-size + 6 * $gap-size;
+  padding: 10px;
+
+  @for $day from 28 through 30 {
+    &--month-length--#{$day} :nth-child(n+#{$day+1}) {
+      display: none;
+    }
+  }
+
+  $days-of-week:
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
+
+  @each $start-day, $start-day-value in $days-of-week {
+    &--start-day--#{$start-day} :first-child {
+      margin-left: $start-day-value * ($day-size + $gap-size);
+    }
+  }
+}
+
+.calendar__cell {
```

Comment:
Use SASS nesting. It should look like:
.calendar [your main BEM block { 
    &__cell [main BEM block's childern] {
    }
}

Checklist: checkpoint 4

Reply:
I totally agree with you, I missed this moment, thanks 🤩